tox.ini

```diff
@@ -10,5 +10,5 @@ setenv =
 deps =
     .[all]
 commands =
-    py.test --cov={envsitepackagesdir}/ebb_lint --cov-report= -rxs [] {envsitepackagesdir}/ebb_lint
+    py.test {posargs} --cov={envsitepackagesdir}/ebb_lint --cov-report= -rxs [] {envsitepackagesdir}/ebb_lint
```

Comment:
Finally got someone to reply to me on this. `[]` is just for backward compatibility for versions of tox that are hella old now. `{posargs}` is good.